testhelpers/testserver/test_api.go

```diff
@@ -0,0 +1,74 @@
+package testserver
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/phayes/freeport"
+	"github.com/stretchr/testify/require"
+
+	"github.com/spacemeshos/explorer-backend/api"
+	"github.com/spacemeshos/explorer-backend/storage"
+	"github.com/spacemeshos/explorer-backend/testhelpers/testutils"
+)
+
+const (
+	testAPIServiceDB = "explorer_test"
+)
+
+// TestAPIService wrapper over fake api service.
+type TestAPIService struct {
+	Storage *storage.Storage
+	server  *api.Server
+	port    int
+}
+
+// StartTestAPIService start test api service.
+func StartTestAPIService(dbPort int) (*TestAPIService, error) {
+	appPort, err := freeport.GetFreePort()
+	if err != nil {
+		return nil, err
```

Comment:
noticed sometimes we use `fmt.Errorf`, sometimes not. Do we have a rule for this? :)